test/package.json

```diff
@@ -7,7 +7,7 @@
     "analysis.build-and-analyze": "npm run build && npm run analysis",
     "build": "npm run build.browser-compile && npm run build.hello-world && npm run build.hello-vdom && npm run build.todo && npm run build.end-to-end && npm run build.ionic",
     "build.browser-compile": "cd ./browser-compile && npm run build",
-    "build.end-to-end": "cd ./end-to-end && npm run build",
+    "build.end-to-end": "cd ./end-to-end && npm ci && npm run build",
```

Comment:
In the old CI, we would have installed these from a previous, unrelated step. This ensures we always have these deps